libclamunrar/dll.cpp

```diff
@@ -230,7 +230,7 @@ int PASCAL RARReadHeaderEx(HANDLE hArcData,struct RARHeaderDataEx *D)
       // open callback for RAR5 archives and if password is invalid.
       if (Data->Arc.FailedHeaderDecryption)
         return ERAR_BAD_PASSWORD;
-
+      
```

Comment:
Seeing multiple places with spaces at the end of lines, did clang-format leave that in?

Reply:
libclamunrar isn't our code, so we don't reformat it, and my text editor hasn't had a chance to their extra whitespace.